app/scripts/views/elements/coppa-age-input.js

```diff
@@ -19,6 +19,11 @@ define(function (require, exports, module) {
       const isRequired = typeof this.attr('required') !== 'undefined';
       const value = this.val();
 
+      const maxValidAge = 130;
```

Comment:
Can you extract this a constant to [just below AuthErrors](https://github.com/mozilla/fxa-content-server/pull/6998/files#diff-9617e8f25f3a7d10987ff2be9db46672R11) so that it's easy to find if we ever want to make updates?